### DISCUSSION TITLE
Sepolia testnet Faucet not sending ETH

### DISCUSSION THREAD

**Discussion by shadowalpha197** [Cyfrin/foundry-full-course-cu#1673] [Q&A] [answered]
I tried using **faucets.chain.link** to get SepoliaETH. I connected my wallet, connect to Github as well to confirm i'm human. But when I _send request_ it says that 'This request doesn't look human to us'. I've refreshed the page multiple times and tried, but it still doesn't seem to work and says the same thing. What can I do? Thanks.

**Comment by EngrPips** [comment, marked as answer]
Hello @shadowalpha197, It seems there is an issue with the chainlink faucet at the moment because I also tried on my end, and I got an error. Below is a picture that shows the response I got back when I tried.

<img width="1440" alt="Screenshot 2024-05-03 at 12 57 41" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/0384b655-68a9-49f2-9b70-85f9ce4b8058">

I will suggest you try other faucets to get sepolia ETH, Below is a list of alternatives.

- https://www.alchemy.com/faucets/ethereum-sepolia
- https://sepolia-faucet.pk910.de/
- https://chainstack.com/sepolia-faucet/
- https://faucet.quicknode.com/ethereum/sepolia

**Reply by shadowalpha197** [reply]
Thank you!

**Reply by EngrPips** [reply]
Anytime. I hope you was able to get the sepolia eth.
